### DISCUSSION TITLE
Post installation settings (Docker)

### DISCUSSION THREAD

**Discussion by phandja** [WeblateOrg/weblate#7197] [General]
Morning,

I have Weblate installed via docker, and everything is almost done. But I still having some issues to finalize my installation:
- Is there any command I can use to reset de git in bulk and not project by project? 
- What are the correct permission for the _data Directory? My installation is working only with 777, which bad, and with the root user as owner.

Regards,

**Comment by nijel** [comment]
The directory should be owned by the user running Weblate. It's UID 1000 in the container. Unless you manually do something with it, it should be created correctly...
